nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/repository/FlowFileEventRepository.java

```diff
@@ -51,4 +51,32 @@
      * @param componentIdentifier Identifier of the component
      */
     void purgeTransferEvents(String componentIdentifier);
+
+    /**
+     * Returns the total number of bytes read by this instance (at the root process group level) since the instance started
```

Comment:
@mattyb149 this is interface I meant to reference in my comments above.

Reply:
Sounds good, will update to return a FlowFileEvent